projects/core/src/site-context/services/site-context-params.service.spec.ts

```diff
@@ -90,4 +100,48 @@ describe('SiteContextParamsService', () => {
     service.setValue(LANGUAGE_CONTEXT_ID, 'en');
     expect(mockLanguageService.setActive).toHaveBeenCalledWith('en');
   });
+
+  describe('getValues', () => {
+    afterAll(() => {
+      // set to initial mock values
+      mockLanguageService.getActive = createSpy().and.returnValue(of('de'));
+      mockCurrencyService.getActive = createSpy().and.returnValue(of(''));
+    });
+
+    it('should emit empty array for empty context', () => {
+      service
+        .getValues([])
+        .pipe(take(1))
+        .subscribe(value => expect(value).toEqual([]));
+    });
+
+    it('should emit active values for single service', () => {
+      mockLanguageService.getActive = createSpy().and.returnValue(
+        of('en', 'de', 'en')
+      );
+      service
+        .getValues([LANGUAGE_CONTEXT_ID])
+        .pipe(bufferCount(3), take(1))
+        .subscribe(values => {
+          expect(values).toEqual([['en'], ['de'], ['en']]);
+        });
+    });
+
+    it('should emit active values for multiple services', () => {
+      mockLanguageService.getActive = createSpy().and.returnValue(of('ja'));
+      mockCurrencyService.getActive = createSpy().and.returnValue(
+        of('', 'USD', 'JPY', 'USD')
```

Comment:
Why we need `''` at the beginning?

Reply:
Wanted to test the functionality that only after all context are set it would emit values.